src/why3prop.ml

```diff
@@ -0,0 +1,127 @@
+(** Translation of lambdapi goals to why3 goals in propositional case *)
+
+open Terms
+open Extra
+
+exception NoGoalTranslation
+
+(* a map from lambdapi terms to Why3 constants *)
+type cnst_table = (term * Why3.Term.lsymbol) list
```

Comment:
For future: improve by using TrmMap.t where TrmMap=Map.Make(Trm) and Trm=struct type t = term let compare = some total comparison function on terms to be defined?